packages/blunderbuss/src/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+declare module "promise-events" {
```

Comment:
do we need this?

Reply:
yes, again this is from probot's janky types.

Reply:
```bash
node_modules/probot/lib/application.d.ts:5:30 - error TS7016: Could not find a declaration file for module 'promise-events'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/promise-events/emitter.js' implicitly has an 'any' type.
  Try `npm install @types/promise-events` if it exists or add a new declaration (.d.ts) file containing `declare module 'promise-events';`

5 import { EventEmitter } from 'promise-events';
                               ~~~~~~~~~~~~~~~~
```